index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>my mobile porifolio</title>
+</head>
+<body>
+    <div class="container">
+        <div class="menu">
+            <div class="logo"><h2><span>My Logo</span></h2></div>
+            <div class="menu-bar"><i class="fa fa-bars fa-2x"></i>
+            </div>
+        <main>
+            <section class="infoText">
+                <h1>I’m Wolter
+                    Glad to see you!</h1>
+                    <p>I’m a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience!
+                         If you like what you see and have a project you need coded,
+                         don’t hestiate to contact me.</p>
+                         <h2>LET'S CONNECT</h2>
+                         <div class="social-media">
```

Comment:
- [ ] You are supposed to use a list to display these icons. Kindly wrap them with the `ul` element and display each of them as a list item using the `li` tag.